templates/conf/quorum.conf.erb

```diff
@@ -1,7 +0,0 @@
-# quorum conf
```

Comment:
Looks like this config is incorporated into zoo.cfg
These commits are upstreams commits.